src/Clean.Architecture.Web/Program.cs

```diff
@@ -47,12 +53,55 @@
   config.Path = "/listservices";
 });
 
+var assemblies = new[]
+{
+  Assembly.GetAssembly(typeof(Contributor)),
+  Assembly.GetAssembly(typeof(CreateContributorCommand))
+};
 
-builder.Host.ConfigureContainer<ContainerBuilder>(containerBuilder =>
+var mediatrOpenTypes = new[]
 {
-  containerBuilder.RegisterModule(new DefaultCoreModule());
-  containerBuilder.RegisterModule(new AutofacInfrastructureModule(builder.Environment.IsDevelopment()));
-});
+  typeof(IRequestHandler<,>),
+  typeof(IRequestExceptionHandler<,,>),
+  typeof(IRequestExceptionAction<,>),
+  typeof(INotificationHandler<>),
+};
+
+foreach (var assembly in assemblies)
+{
+  foreach (var openInterfaceType in mediatrOpenTypes)
+  {
+    if (assembly != null)
+    {
+      assembly.GetTypes()
+        .Where(t => t.IsClass && !t.IsAbstract && t.GetInterfaces().Any(i => i.IsGenericType && i.GetGenericTypeDefinition() == openInterfaceType))
+        .ToList()
+        .ForEach(implementationType =>
+        {
+          var serviceType = implementationType.GetInterfaces().First(i => i.IsGenericType && i.GetGenericTypeDefinition() == openInterfaceType);
+          builder.Services.AddScoped(serviceType, implementationType);
+        });
+    }
+  }
+}
```

Comment:
Can this loop be removed in favor of using MediatR's own DI registration method?

```
builder.Services.AddMediatR(cfg => cfg.RegisterServicesFromAssemblies(assemblies));
```